templates.xml

```diff
@@ -55,6 +69,20 @@
     <properties key="filters" value="Project\ Difficulty|Beginner"/>
     <properties key="stockConfigCompatibility" value="com.silabs.ss.framework.project.toolchain.core.default"/>
   </descriptors>
+  <descriptors name="rm1262_class_a_env" label="RM1262 - Class A Environmental data"
+               description="This example project shows a Class A connection being established and temperature, humidity and air quality data being uplinked from a MikroE 4558 board.">
+    <properties key="namespace" value="template.uc"/>
+    <properties key="keywords" value="universal\ configurator"/>
+    <properties key="projectFilePaths" value="app/example/rm1262_class_a_env/rm1262_class_a_env.slcp"/>
```

Comment:
should this have the example directory? I think they were removed before?

Reply:
This path is OK, the removed example path was for docs that have been moved into one folder.